### PR TITLE
Line graph - for constant time & Labels for empty graph

### PR DESCRIPTION
**Pull request by Divyanshu11011**
# Description
Added Line graph - for constant time & Labels for empty graph

# Snapshot

image

### CODE REVIEW

**Comment by sunitakawane** [comment]
Screenshot 2024-07-16 at 4 54 51 PM
When we toggle `one graph per query` nothing shows up

**Comment by Divyanshu11011** [comment]
image
Fixed that 👍

File: static/js/metrics-explorer.js [hunk 1]
```diff
@@ -1432,137 +1537,113 @@ $('#json-block').on('click', function() {
 });
 
 // Merge Graphs in one
-function mergeGraphs(chartType, panelId=-1) {
-    if(isDashboardScreen){
```
Comment (sunitakawane) [inline static/js/metrics-explorer.js:1436]: why did you remove this condition. This is being used in dashboards when we use metrics explorer.
